.github/workflows/ci.yml

```diff
@@ -503,6 +503,13 @@ jobs:
             echo "No trailing commas for function calls"
             exit 1
           fi
+      - name: Check for empty string comparison
+        run: |
+          # Using `grep .` as the exit codes are always true for correct grammar
+          if tree-grepper -q go '((binary_expression (identifier) ["==" "!="] (interpreted_string_literal) @ri) (#eq? @ri "\"\""))' | grep .; then
+            echo "Do not compare with empty string"
```

Comment:
nit: could you please change this to be something like `use len check instead of empty string comparison` so that folks know what the expectation is?